README.md

```diff
@@ -1,10 +1,116 @@
 # microsoft-sql-server-2017-cis-baseline
-An InSpec Profile to validate the Microsoft SQL Server 2017 CIS Benchmark, version 1.2.0.
----
-Name: microsoft-sql-server-2017-cis-baseline
-Title: microsoft-sql-server-2017-cis-baseline
-Maintainer: MITRE SAF
-Copyright: MITRE SAF
-Copyright Email: saf@groups.mitre.org
-License: Apache-2.0
-Version: 1.2.0
+An InSpec Profile to validate the secure configuration of Microsoft SQL Server 2017 CIS Benchmark, against [CIS](https://www.cisecurity.org/cis-benchmarks/)'s CIS Microsoft SQL Server 2017 version 1.2.0.
+
+## Getting Started  
+
+__For the best security of the runner, always install on the runner the _latest version_ of InSpec and supporting Ruby language components.__ 
+
+Latest versions and installation options are available at the [InSpec](http://inspec.io/) site.
+
+## Tailoring to Your Environment
+The following inputs must be configured in an inputs ".yml" file for the profile to run correctly for your specific environment. More information about InSpec inputs can be found in the [InSpec Profile Documentation](https://www.inspec.io/docs/reference/profiles/).
+
+```yaml
+# Username MSSQL DB Server
+user: ''
+
+# Password MSSQL DB Server
+password: ''
+
+# Hostname of MSSQL DB Server
+host: ''
+
+# Instance name of MSSQL DB Server
+instance: ''
+
+# Port for MSSQL DB Server (numeric)
+port: 
+
+# Approved version expected to be installed (numeric)
+approved_sql_version: 
+
+# Set to True if the MS SQL Server instance is clustered
+clustered_instance: False
+
+# Enter the maximum number of error log files allowed
+max_error_logs: 12
```

Comment:
Since we're only offering this as an input for convenience during overlay development, should this be listed on the README? [The control](https://github.com/mitre/microsoft-sql-server-2017-cis-baseline/blob/main/controls/microsoft-sql-server-2017-cis-5.1.rb) states that the value should be 12 or higher to be compliant.

Reply:
I left it in the README, but with a further comment that clarifies why it's an input (i.e. to allow for organizations to tailor the value to their own requirements).

Reply:
Thanks @wdower 
This is helpful!